src/libs/sfdk/utils.cpp

```diff
@@ -88,4 +88,18 @@ Utils::PortList toPortList(const QList<quint16> &portList)
     return toPortList(Utils::transform(portList, [](quint16 port) { return Port(port); }));
 }
 
+QString separator(TextStyle textStyle)
+{
+    switch (textStyle) {
+        case TextStyle::Pretty:
+            return {QChar::Nbsp};
+        case TextStyle::CamelCase:
+            return {};
+        case TextStyle::SnakeCase:
+            return QStringLiteral("_");
+    }
+    return {};
+}
+
+
```

Comment:
Extra blank line.

Now that you are here once again, we can also use `return {'_'};` for the SnakeCase case above :)

Reply:
Fixed